packages/components/src/internal/components/forms/FieldLabel.tsx

```diff
@@ -83,7 +83,7 @@ export class FieldLabel extends Component<FieldLabelProps> {
                         inputFieldName={getFieldEnabledFieldName(column, fieldName)}
                         on={toggleProps && toggleProps.on ? toggleProps.on : 'Enabled'}
                         off={toggleProps && toggleProps.off ? toggleProps.off : 'Disabled'}
-                        style={style ? style : { float: 'right' }}
+                        style={style ? style : { float: 'right', width: 90, height: 28 }} /* fixed width and height needed for non-formsy inputs */
```

Comment:
Can't we know if this is formsy or not? `labelOverlayProps?.isFormsy`?  Regardless, we should roll these styles into a class. Something about all of this doesn't seem right...

Reply:
switched to use class. Though the class solution is not pretty either: add to add !important to the width height since a bad inline style width/height is added automatically.